src/commonMain/kotlin/uk/gov/hmrc/calculator/utils/validation/PensionValidator.kt

```diff
@@ -17,14 +17,47 @@ package uk.gov.hmrc.calculator.utils.validation
 
 import uk.gov.hmrc.calculator.model.TaxYear
 import uk.gov.hmrc.calculator.model.pension.PensionAllowances.getPensionAllowances
+import uk.gov.hmrc.calculator.utils.validation.HoursDaysValidator.isTwoDecimalPlacesOrFewer
+import kotlin.jvm.JvmSynthetic
 
 object PensionValidator {
 
-    fun isValidYearlyPension(yearlyWage: Double, yearlyPension: Double): Boolean {
+    fun isValidYearlyPension(yearlyPension: Double, wage: Double, taxYear: TaxYear): MutableList<PensionError> {
```

Comment:
I think we can keep the naming clarity that it is yearly wage here still to avoid confusion, as pension is usually dealth with in months.

```suggestion
    fun isValidYearlyPension(yearlyPension: Double, yearlyWage: Double, taxYear: TaxYear): MutableList<PensionError> {
```

Is this function going to be used by the external consumers? If so, is it worth/is there need for having a `isValidMonthlyPension` validator too that takes monthly pension and wage as parameters to do the conversion to annual inside the function, rather than needing to convert beforehand?